substrate/frame/revive/src/storage/meter.rs

```diff
@@ -482,8 +482,7 @@ impl<T: Config, E: Ext<T>> RawMeter<T, E, Nested> {
 	///
 	/// We normally need to call this **once** for every call stack and not for every cross contract
 	/// call. However, if a dedicated limit is specified for a sub-call, this needs to be called
-	/// once the sub-call has returned. For this, the [`Self::enforce_subcall_limit`] wrapper is
-	/// used.
+	/// once the sub-call has returned.
```

Comment:
Doc comment needs updating to reflect the new much simpler behavior: This has to be called every time a sub call returns. No special casing anymore for a 'dedicated limit'. We just removed this special casing.

Reply:
This comment has been changed on my tree, and no longer contains any reference to `enforce_subcall_limit`.

Reply:
The rest of the comment is still stale:
```
/// We normally need to call this **once** for every call stack and not for every cross contract
/// call. However, if a dedicated limit is specified for a sub-call, this needs to be called
```

There is no special limit anymore. Also, we never call it once per call stack as stated here. We call it for every sub call now.

Reply:
😵

I was wrong, thanks for clarifying.